Serverless AWS-CDK Node.js instrumentation change (#9759)
DataDog/documentation: content/en/serverless/installation/nodejs.md

{{% /tab %}}
{{% tab "AWS CDK" %}}

The [Datadog CDK Constructs][1] automatically configures ingestion of metrics, traces, and logs from your serverless applications by:

- Installing and configuring the Datadog Lambda library for your Python and Node.js Lambda functions.
- Enabling the collection of traces and custom metrics from your Lambda functions.
- Managing subscriptions from the Datadog Forwarder to your Lambda function log groups.

### Install the Datadog CDK Constructs Library

> Run the following Yarn or NPM command in your CDK project

Reviewer: ```suggestion
Run the following Yarn or NPM command in your CDK project:
```

```sh
#Yarn
yarn add --dev datadog-cdk-constructs

#NPM
npm install datadog-cdk-constructs --save-dev
```

The Datadog CDK Constructs library is now downloaded and ready to use.

### Instrument the function

> Import the `datadog-cdk-construct` module in your AWS CDK app and add the following configurations. See the sample code below in TypeScript (the usage in other language should be similar).

Reviewer: ```suggestion
Import the `datadog-cdk-construct` module in your AWS CDK app and add the following configurations (this example is TypeScript, but usage in other languages is similar):
```
